modules/configuring-otlp-output.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging-6.0/log6x-output.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-otlp-output_{context}"]
+= Configuring OTLP output
+
+The OpenTelemetry Protocol (OTLP) allows the cluster administrator to collect and forward logs to the OTLP receivers. The OTLP sends data over HTTP using JSON encoding by adhering to the https://opentelemetry.io/docs/specs/otlp/[OpenTelemetry Observability Framework].
+
+:FeatureName: The OTLP output log forwarder
+include::snippets/technology-preview.adoc[]
+
+.Procedure
+
+* To enable OTLP, create or edit the `ClusterLogForwarder` CR using the template below:
++
+.Example ClusterLogForwarder CR
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+    name: my-logforwarder
+    namespace: my-app-namespace
+    annotations: # <1>
+        observability.openshift.io/tech-preview-otlp-output: "enabled"
+spec:
+    outputs:
+    - type: otlp # <2>
+      name: otel-collector
+      tls:
+        insecureSkipVerify: true # <3>
```

Comment:
We shouldn't provide an example that goes against good practices/security and this is one. Nobody in a productive environment should set `insecureSkipVerify: true`

Reply:
+ 1 to Oscar's point, @smunje1 why don't we change the value in the example to false & mention that it can be set to true for test/dev environments but is not recommended or supported for production?

Reply:
How about the following OTLP example?

```
apiVersion: observability.openshift.io/v1
kind: ClusterLogForwarder
metadata:
  annotations:
    observability.openshift.io/tech-preview-otlp-output: "enabled"
  name: clf-otlp
spec:
  outputs:
  - name: otlp
    type: otlp
    otlp:
      tuning:
        compression: gzip
        delivery: atLeastOnce
        maxRetryDuration: 20
        maxWrite: 10M
        minRetryDuration: 5
      url: <otlp-url>
  pipelines:
  - inputRefs:
    - application
    - infrastructure
    - audit
    name: otlp-logs
    outputRefs:
    - otlp
  serviceAccount:
    name: logging-otlp
```

Reply:
Reference: https://gitlab.cee.redhat.com/aosqe/aosqe-tools/-/blob/master/logging/log_doc/clf-6.0-examples/otlp.yaml?ref_type=heads

Reply:
I will add callouts once we are ok with OTLP example.

Reply:
Please see my comment: https://github.com/openshift/openshift-docs/pull/78924/files#r1715505169